src/index.html

```diff
@@ -5,9 +5,64 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link
+    rel="icon"
+    type="image/x-icon"
+    href="images/star-active.svg"
+    >
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="card">
+
+      <img
+
+        class="card__good"
+
+        src="./images/imac.jpeg"
+
+        alt="imac"
+
+      >
```

Comment:
```suggestion
    <div data-qa="card" class="card">
      <img
        class="card__good"
        src="./images/imac.jpeg"
        alt="imac"
      >
```

Too mane blank lines